### DISCUSSION TITLE
Should we rename our repository to prime-reportstream?

### DISCUSSION THREAD

**Discussion by RickHawesUSDS** [CDCgov/prime-reportstream#1201] [Ideas]
Should we rename our repository to `prime-reportstream`? 

**Implications**
- GitHub will redirect all references to https://github.com/cdcgov/prime-data-hub to https://github.com/cdcgov/prime-reportstream. 
- Members of the team should set their origins to the new URL by doing a `git remote set-url  https://github.com/cdcgov/prime-reportstream`

**Comment by carlosfelix2** [comment]
Might as well keep the names straight.  I say AYE!

**Comment by MauriceReeves-usds** [comment]
From a branding point of view I think it makes sense to change our repo name, but I think we should recognize that it could cause some issues for devs and be prepared to handle that.

**Comment by drew-usds** [comment]
💯 let's do it!

There are also a lot of references to `data-hub` in the router itself ... that might be a _separate_ discussion though.

**Comment by RickHawesUSDS** [comment]
Done. 

```
git remote set-url origin https://github.com/cdcgov/prime-reportstream
```
